Sources/BadondeCore/BurghCommand.swift

```diff
@@ -154,23 +156,35 @@ class BurghCommand: Command {
 			!rawMilestone.isEmpty
 		{
 			Logger.step("Fetching repo milestones for '\(repositoryShorthand)'")
-			let milestones = try milestoneAPI.getMilestones(for: repositoryShorthand).map { $0.title }
-			if let milestone = milestones.fuzzyMatch(word: rawMilestone) {
+			let milestones = try milestoneAPI.getMilestones(for: repositoryShorthand)
+			let milestoneTitles = milestones.map { $0.title }
+			if
+				let milestoneTitle = milestoneTitles.fuzzyMatch(word: rawMilestone),
+				let milestone = milestones.first(where: { $0.title == milestoneTitle })
+			{
```

Comment:
<!--
  1 failure:  Opening braces sh...
  0 warning: 
  
  
  DangerID: danger-id-default;
  File: Sources/BadondeCore/BurghCommand.swift;
  Line: 164;
-->
- :no_entry_sign: Opening braces should be preceded by a single space and on the same line as the declaration. (`opening_brace`)